friends/src/App.js

```diff
@@ -51,7 +56,7 @@ class App extends Component {
           </div>
           ))}
         </ul>
-      <Form onClick={this.state.addFriend} value={this.state.inputHandler}/>
+      <Form handleSubmit={this.addFriend} value={this.inputHandler}/>
```

Comment:
Line 59: In Form.js, you're trying to access `props.friend`, but on this line here, you aren't passing in anything as `friend` into props.

Try:
```
<Form
  friend={this.state.friend}
  handleSubmit={this.addFriend}
  value={this.inputHandler}
/>
```

This will allow you to access `props.friend` inside of the Form component.